src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -17,6 +17,11 @@ export class VeteranBenefitSummaryLetter extends React.Component {
   }
 
   handleChange(domEvent) {
+    window.dataLayer.push({
+      event: 'letter-benefit-option-clicked',
+      'letter-benefit-option': domEvent.target.id,
```

Comment:
The pushes just queue objects locally that eventually get sent to the Google servers for processing where the actual "state" resides.

So if we had a tag that reported out something like `Category: Interaction`, `Action: {{ letter-benefit-option }} - {{ letter-benefit-option-clicked }}`, `Label: {{ Event }}` into GA, the value of those variables `letter-benefit-option` and `letter-benefit-option-clicked` is stored on the GA side.

If events like the below were pushed locally:

```
{
  event: 'letter-benefit-option-clicked',
  'letter-benefit-option': 'foo',
  'letter-benefit-option-status': 'checked'
}

{
  event: 'letter-benefit-option-clicked',
  'letter-benefit-option': 'foo',
}

{
  event: 'letter-benefit-option-clicked',
  'letter-benefit-option': 'foo',
  'letter-benefit-option-status': 'unchecked'
}
```

The event actions reported in GA should be:

```
foo - checked
foo - checked
foo - unchecked
```

If those event pushes arrived out of order at the GA servers with the second one arriving last (because the internet), you'd get:

```
foo - checked
foo - unchecked
foo - unchecked
```

Which is what that comment is meant to warn you about. Once you overwrite the value of variable on the GA server, it'll persist just that new value. So, if your actual reporting of an event depends on a variable, you should always push them together to be sure the variable has the correct value at processing time.

Hope that clarifies!